src/Subscript.h

```diff
@@ -30,7 +30,7 @@
 template <typename T, size_t size>
 struct index_array
 {
-	T ix_[size];
+	T ix_[size+1];
```

Comment:
Actually, just noticed this. This is a change from one of my branches. Did you make this in your branch ?

Reply:
I don't recall making this change.

Reply:
May have been a thing that got introduced during rebase. It is the right change, and PR #88 already had this.